tests/Facebook/InstantArticles/AMP/AMPArticleTest.php

```diff
@@ -395,25 +372,6 @@ public function testImageHeightDownloadDisabled()
         $this->assertEquals($expectedHeight, $firstArticleImageElement->getAttribute('height'));
     }
 
-    public function testImageHeightDownloadEnabled()
```

Comment:
why this test was removed? shouldn't we have a test case for ENABLE_DOWNLOAD_FOR_MEDIA_SIZING_KEY = true and other for the media cache?

If travis can't access the internet, we should at least have the cache dir test